components/TravelPlansList/index.js

```diff
@@ -0,0 +1,37 @@
+import useSWR from 'swr';
+
+const fetcher = async (url) => {
+  const res = await fetch(url);
+  const data = await res.json();
+  return data;
+};
+
+const TravelPlansList = ({ travelPlans }) => {
```

Comment:
```suggestion
const TravelPlansList = () => {
```

Reply:
The `travelPlans` prop could be removed, since you now fetch the data with `useSWR`, or?